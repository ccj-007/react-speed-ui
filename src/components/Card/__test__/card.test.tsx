import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Card } from '../../index';

describe('test Card component', () => {
  it('should render', () => {
    const { asFragment } = render(<Card />);
    expect(asFragment()).toMatchSnapshot();
  });
});
