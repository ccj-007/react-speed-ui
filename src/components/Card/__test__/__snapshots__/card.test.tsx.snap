// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Card component should render 1`] = `
<DocumentFragment>
  <div
    class="speed-card css-1lj97fx-Card"
  >
    <div
      class="speed-card-header"
    >
      <div>
        default header title
      </div>
       
    </div>
    <div
      class="speed-card-content"
    />
    <div
      class="speed-card-footer"
    >
      default footer title
    </div>
  </div>
</DocumentFragment>
`;
